### DISCUSSION TITLE
Getting error of 401 Login Required

### DISCUSSION THREAD

**Discussion by sanjaydeveloper15** [freekmurze/laravel-analytics#430] [Q&A]
Saying expected oauth2 access token but as per README there is no option anywhere where need to give it.

I added json file and view I'd apart from it where I need to add oath 2 access token can someone please tell me

**Comment by warmwhisky** [comment]
Just started getting this today. No clue what I am supposed to do now as the JS library appears to be getting deprecated March 31 2023
https://developers.google.com/identity/sign-in/web/sign-in
